user/templates/user/data_transfer.html

```diff
@@ -0,0 +1,54 @@
+{% endblock style %}
+
+{% block content %}
+<h1>Data Transfer</h1>
+    <div class="card">
+        <div class="card-header">
+            <!-- Nav tabs -->
+            <ul class="nav nav-tabs" role="tablist">
+            <li class="nav-item">
+                <a class="nav-link active" data-bs-toggle="tab" href="#iCommands">iCommands</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" data-bs-toggle="tab" href="#Gocommands">Gocommands</a>
+            </li>
+            </ul>
+        </div>
+        <div class="card-body">
+            <!-- Tab panes -->
+            <div class="tab-content">
+                <div id="iCommands" class="container tab-pane active"><br>
+                    <p>
+                        Yoda is based on iRODS technology. The iCommands allows you to interact with the iRODS backend of Yoda directly.
+                        Therefore, it is possible to transfer data to and from Yoda using the iRODS communication protocol.
+                        You need to configure the iCommands to connect to this Yoda environment.
+                    </p>
+                    <p> Copy and paste the following configuraton into your ~/.irods/irods_environment.json configuration file. </p>
+                    <pre><code id="code-block1">{&#10;    "irods_host": "{{ config.get('IRODS_ICAT_HOSTNAME') }}",&#10;    "irods_port": 1247,&#10;    "irods_home": "/{{ config.get('IRODS_DEFAULT_ZONE') }}/home",&#10;    "irods_user_name": "{{ g.user }}",&#10;    "irods_zone_name": "{{ config.get('IRODS_DEFAULT_ZONE') }}",&#10;    "irods_authentication_scheme": "pam",&#10;    "irods_encryption_algorithm": "AES-256-CBC",&#10;    "irods_encryption_key_size": 32,&#10;    "irods_encryption_num_hash_rounds": 16,&#10;    "irods_encryption_salt_size": 8,&#10;    "irods_client_server_negotiation": "request_server_negotiation",&#10;    "irods_client_server_policy": "CS_NEG_REQUIRE"&#10;}</code></pre>
+                    <div class="float-md-end">
+                        <button id="download-button1" type="button" class="btn btn-outline-secondary btn-download-file"> Download as JSON </button>
+                    </div>
+                </div>
+                <div id="Gocommands" class="container tab-pane fade"><br>
+                    <p>Gocommands can read configuration from a YAML file. </p>
+                    <p>Copy and paste the following configuration into your config.yml file.</p>
+                    <pre><code id="code-block2">irods_host: "{{ config.get('IRODS_ICAT_HOSTNAME') }}"&#10;irods_port: 1247&#10;irods_user_name: "{{ g.user }}"&#10;irods_zone_name: "iplant"&#10;irods_user_password: "your password"</code></pre>
```

Comment:
We can prefill the irods_zone_name with "{{ config.get('IRODS_DEFAULT_ZONE') }}". We do not want to have the pasowrd in the config file, so we can remove that line.